pkg/ipcache/cidr.go

```diff
@@ -36,7 +36,7 @@ import (
 //
 // Deprecated: Prefer UpsertLabels() instead.
 func (ipc *IPCache) AllocateCIDRs(
-	prefixes []netip.Prefix, oldNIDs []identity.NumericIdentity, newlyAllocatedIdentities map[netip.Prefix]*identity.Identity,
+	prefixes []netip.Prefix, newlyAllocatedIdentities map[netip.Prefix]*identity.Identity,
```

Comment:
Please update the function doc regarding `oldNIDs`.

Reply:
(nice cleanup by the way, I always felt a little weird about this parameter to this function)